app-web/plugins/gatsby-source-github-all/utils/plugins.js

```diff
@@ -84,30 +90,31 @@ const markdownFrontmatterPlugin = (extension, file) => {
  * @param {Object} file
  * @returns {Object} the modified file
  */
-const markdownPagePathPlugin = (extension, file) => {
-  if (extension !== 'md') {
+const pagePathPlugin = (extension, file) => {
+  if (extension !== 'md' && extension !== 'html') {
     return file;
   }
   // check front matter for a resourcePath
-  const data = matter(file.content);
-  const frontmatter = data.data;
-  if (frontmatter.resourcePath) {
-    file.metadata.resourcePath = frontmatter.resourcePath;
-  } else {
-    // no resource path, this file is destined to be turned into a page,
-    // the page page is composed of the source name, the title of the file plus an id
-    file.metadata.resourcePath = createPathWithDigest(
-      file.metadata.source,
-      file.metadata.source,
-      file.metadata.name,
-      file.html_url
-    );
+  if (extension === 'md') {
+    const data = matter(file.content);
+    const frontmatter = data.data;
+    if (frontmatter.resourcePath) {
+      file.metadata.resourcePath = frontmatter.resourcePath;
+      return file;
+    }
   }
-
+  // no resource path, this file is destined to be turned into a page,
+  // the page page is composed of the source name, the title of the file plus an id
+  file.metadata.resourcePath = createPathWithDigest(
+    file.metadata.source,
+    file.metadata.source,
```

Comment:
duplication?

Reply:
thats not a duplication, this fn is called `createPathWithDigest(baseURL, ...digestables)`
should I comment that?